.github/workflows/benchmark.yml

```diff
@@ -30,8 +30,8 @@ jobs:
       - name: Checkout CredData
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
-          repository: Samsung/CredData
-          ref: main
+          repository: babenek/CredData
```

Comment:
TODO: rollback after https://github.com/Samsung/CredData/pull/187
and fix style